### ISSUE TITLE
nixos.tests.hibernate.x86_64-linux is failing in release-nixos-17.09

### ISSUE THREAD

**Issue by symphorien** [NixOS/nixpkgs#29054]
## Issue description

channels/nixos-17.09 seems to be blocked by hibernation tests (https://hydra.nixos.org/job/nixos/release-17.09/tested#tabs-constituents) so I tried to find the cause of this failure. I focused on the first failing build (https://hydra.nixos.org/build/59947774#tabs-summary). I can reproduce the failure locally, and if I revert a55be749fdbded85fe95bad8c3251baf8882695c (netcat-openbsd: 1.105 -> 1.130) test passes again, but I don't understand why.

What I observed is that probe is indeed sending "test" to machine, machine is indeed receiving it, but never closes the connection (no ACK ?). I have been unable to use tshark in the test vm.

### Steps to reproduce

in nixpkgs:

```
git checkout 67c46eff0094ddcad56e3bf9d0ed924ad27876fd
nix-build "nixos/release-combined.nix" -A 'nixos.tests.hibernate.x86_64-linux'
# hangs
git revert a55be749fdbded85fe95bad8c3251baf8882695c
nix-build "nixos/release-combined.nix" -A 'nixos.tests.hibernate.x86_64-linux'
# succeeds
```
## Technical details

* System:17.09beta41.1b8c7786ee (Hummingbird)
* Nix version:  1.11.14
* Nixpkgs version: "17.09beta43.0fd7ef61b2"
* Sandboxing enabled: yes

**Comment by NeQuissimus** [comment]
I fixed the test in 024b501907. `netcat` wants to have a `-q` param now...
Picked into `release-17.09` and `staging` since they both had the `netcat-openbsd` update.

**Comment by NeQuissimus** [comment]
Looks like the channel just got an update, the test passes on Hydra
